addons/udes_stock_package_move/wizard/change_package_location.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from odoo import models, api, fields, _
+
+
+class ChangePackageLocation(models.TransientModel):
+    _name = "change_package_location"
+
+    picking_type_id = fields.Many2one(
+        "stock.picking.type", string="Picking type", required=True,
+        default=lambda self: self._default_picking_type()
+    )
+
+    reference = fields.Char("Reference")
+
+    location_dest_id = fields.Many2one(
+        "stock.location", string="New destination location",
+    )
+
+    def _default_picking_type(self):
+        """ Returns the Internal Transfer picking type """
+        Users = self.env["res.users"]
+        warehouse = Users.get_user_warehouse()
+        return warehouse.int_type_id
+
+    def create_picking(self):
+        Package = self.env["stock.quant.package"]
+
+        self.ensure_one()
+
+        packages = Package.browse(self.env.context["active_ids"])
```

Comment:
What happens if any of the quants of the packages is reserved? Probably we want a check it and raise an error before calling `create_picking`.